web/src/components/pages/datasets/metadata-viewer/types.ts

```diff
@@ -0,0 +1,41 @@
+export type LanguageDataset = {
+  id: number
+  dataset_id: number
+  locale_id: number
+  total_clips_duration: number
+  valid_clips_duration: number
+  average_clips_duration: number
+  total_users: number
+  size: number
+  checksum: string
+  release_date: Date
+  name: string
+  release_dir: string
+  download_path: string
+  splits?: Splits
+}
+
+export type Splits = {
+  gender: Gender
+  age: Age
+}
+
+type Age = {
```

Comment:
If you export `type Age` you could use it as type for your keys in the `AGE_MAPPING` type. That should give you some more type safety.